### DISCUSSION TITLE
Vorta throwing error messages after apparently succesful backups

### DISCUSSION THREAD

**Discussion by alikstimix** [borgbase/vorta#2071] [FAQ]
I upgraded to v 0.8.12 recently and had a few problems with password being rejected.  I fiddled about a bit and managed to reestablish connection to my repositories.

I can now see and mount old archives successfully but any new backup throws an error after  reporting a successful completion.  

borg list shows the new backup, but it is not listed on the Vorta Archives screen.

Similar errors show up when trying to refresh an old archive.
I tried reinstalling vorta, no difference.
I tried deleting settings.db and importing a json profile backup, no difference.

Anyone got any clues as to what is going wrong?

![Screenshot from 2024-09-10 10-49-44](https://github.com/user-attachments/assets/9a1d9d09-bd08-49cd-adad-309146cefdde)

**Comment by m3nu** [comment]
The errors in your screenshot happen in Borg, not Vorta. There are also 2 different Python environments for some reason. Vorta 0.8.12 is also pretty old by now and wasn't tested on Python 3.12. If you use cutting-edge Python, why not use a newer Vorta and Borg version?

**Reply by alikstimix** [reply]
Thanks for the rapid response.
I am just using the latest borg and vorta provided by Mint (Wilma)

I'm not a fan of flatpak, so I guess I need to compile the latest versions?

**Reply by m3nu** [reply]
Or `pip install vorta`? Is nobody using pip any more? 😄

**Reply by alikstimix** [reply]
pip install "borgbackup==1.4.0"
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.

**Reply by alikstimix** [reply]
So what do I do?  uninstall vorta or borgbackup, then pip?
